sbc_translation/models/translation_user.py

```diff
@@ -31,11 +31,10 @@ class TranslationUser(models.Model):
         "res.lang.compassion", domain=[("translatable", "=", True)],
         help="Utility field only used for the search view"
     )
-    search_competence_ids = fields.Many2many(
+    search_competence_id = fields.Many2one(
         "translation.competence",
-        help="Utility field only used for the search view",
-        compute="_compute_competences",
```

Comment:
I don't see the point here. Why do you remove the compute and change to Many2one? The translator can have multiple competences, not only one. If you remove the compute, this field will always be empty.